package.json

```diff
@@ -25,18 +25,18 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-passport",
   "dependencies": {
-    "passport": "~0.4.0"
+    "passport": "^0.5.0"
   },
   "peerDependencies": {
-    "bedrock": "1.12.1 - 3.x",
+    "bedrock": "^4.0.0",
     "bedrock-account": "2.x - 5.x",
```

Comment:
This needs to be updated to the latest too.

Reply:
upgraded to `bedrock-account: ^5.0.0` here: https://github.com/digitalbazaar/bedrock-passport/pull/44/commits/06eaf15926ae97d44136c1e9df56eb8416318bc5